front-end/src/app/tabs/dashboard/dashboard.page.html

```diff
@@ -138,7 +138,7 @@ <h1 *ngIf="!app.isInMobileMode()">{{ 'DASHBOARD.USEFUL_LINKS' | translate }}</h1
             (select)="openUsefulLink(link)"
           >
             <app-statistics-button
-              *ngIf="app.user.isAdministrator && !editMode"
+              *ngIf="(app.user.isAdministrator || app.user.canManageDashboard) && !editMode"
```

Comment:
Lascerei solo agli admin, o dovremmo modificare in modo più pesante il Resource Controller delle statistiche